test/sslapitest.c

```diff
@@ -998,6 +1105,11 @@ static int test_ktls_client_server(void)
     return execute_test_ktls(1, 1, 1, 1);
 }
 
+static int test_ktls_sendfile(void)
+{
+    return execute_test_ktls_sendfile();
```

Comment:
Not sure why you have this intermediate function. Why not just rename `execute_test_ktls_sendfile` to `test_ktls_sendfile`

Reply:
Right, this is just an element of copy-pasting the other ktls tests

Reply:
Fixed.